cmd/run.go

```diff
@@ -562,6 +558,8 @@ func executeRun(runTemplateName, runFilePath string, apps []runfileconfig.App) (
 
 			if runState.AppCMD.Command.Process != nil {
 				putAppProcessIDInMeta(runState)
+				// Attach a job object to the app process.
+				daprsyscall.AttachJobObjectToProcess(strconv.Itoa(os.Getpid())+"-"+utils.WindowsDaprAppProcJobName, runState.AppCMD.Command.Process)
```

Comment:
Can we encapsulate `strconv.Itoa(os.Getpid())+"-"+utils.WindowsDaprAppProcJobName` into a helper function since it's being used at multiple places? Probably as part of utils, and we can unexport `WindowsDaprAppProcJobName`.